src/interface/src/app/plan/plan.component.ts

```diff
@@ -113,4 +129,12 @@ export class PlanComponent implements OnInit, OnDestroy {
   backToOverview() {
     this.router.navigate(['plan', this.plan!.id]);
   }
+
+  goBack() {
+    if (this.showOverview$.value) {
+      this.router.navigate(['home']);
```

Comment:
More of a clarifying question than a comment: Right now if you start on the map => create a planning area => get to the scenario config page => Click on the back arrow, you get sent home rather than back to the map. Is that the desired flow?

Reply:
if you are in a scenario page you go back to the plan page. If you are on the plan page you go back home

Reply:
Ah, sorry mixed up my terms. So even if you're coming from the map, if you're on the plan page then the back arrow is essentially another home button?

Reply:
yes, i can double check if that is the desired behavior on that flow